package.json

```diff
@@ -5,13 +5,17 @@
   "repository": "git://github.com/ianstormtaylor/metascraper.git",
   "license": "MIT",
   "main": "./dist/index.js",
+  "files": [
+    "lib"
+  ],
   "scripts": {
     "build": "npm-run-all build:dist build:tests",
     "build:tests": "browserify ./test/browser.js --transform babelify --outfile ./test/support/build.js",
     "build:dist": "babel ./lib --out-dir ./dist",
     "clean": "rm -rf ./dist ./node_modules ./test/support/build.js",
     "compare": "node ./support/comparison",
     "prepublish": "npm run build:dist",
+    "postinstall":"if ! [ -d ./dist ] ; then npm run build:dist ; fi ",
```

Comment:
I added this so that we can npm install directly from github (basically so I can use this branch). Feel free to remove / change, but I think it could be useful.